### ISSUE TITLE
Good Fences Make Good Neighbors

### ISSUE THREAD

**Issue by RandomDSdevel** [dshadowwolf/ideas-for-a-new-language#11]
@dshadowwolf:  You should make [a GitHub organization](https://help.github.com/articles/about-organizations/) for this repository (or the multiple repositories that should ultimately result out of the resolution of #10.)  That way, it couldn't get lost in among all of your personal repositories.  You'd have to come up with a _name_ for it&thinsp;—&thinsp;probably something along the lines of 'name of language'-lang&thinsp;—, though, so I guess this increases the priority of #1.

**Comment by dshadowwolf** [comment]
At this time a separate org for this doesn't, really, make sense as its just a collection of thoughts and ideas - one that I don't reference frequently because I felt really, really drained after writing the initial collection of documents.

But yes, it probably should, so that tutorials and even implementations of languages using some of the ideas could be done and hosted.

**Comment by RandomDSdevel** [comment]
@dshadowwolf:  Fair enough!  I'll just lurk around here until you feel up to figuring out exactly what structure you want for things, then.  In the meantime, try not to bury this repository (or the multiple ones that might result out of the resolution of #10) under too many other ones, alright?  😜

**Comment by RandomDSdevel** [comment]
@dshadowwolf:  Come to think of it, maybe a Slack channel might come in handy, too.
